lib/ansible/modules/cloud/amazon/cloudfront_facts.py

```diff
@@ -531,6 +531,10 @@ def keyed_list_helper(self, list_to_key):
 
 def set_facts_for_distribution_id_and_alias(details, facts, distribution_id, aliases):
     facts[distribution_id].update(details)
+    # also have a fixed key for accessing results/details returned
+    facts['result'] = details
+    facts['result']['DistributionId'] = distribution_id
```

Comment:
This needs adding to `RETURN` so people know it exists,
Could you please another block like https://github.com/ansible/ansible/blob/b20c04361d17b28c0a1ef3e61b95ab4446365961/lib/ansible/modules/cloud/amazon/cloudfront_facts.py#L193-L198 for `DistributionId`

Also I note that the other variables seem to be lower case with `_`, so to be consistent should this be 

```suggestion
    facts['result']['distribution_id'] = distribution_id
```

Reply:
Probably the docs are wrong (or not matching the code) here. In fact all requrned AWS values are camel case in this module, hence I kept the pattern for now. 

Not sure what's the best strategy here, change the code to be more conventional (snake case) and break backwards compatibility or just update the docs to be in sync with the code.

Reply:
Oh, good spot.
I wonder if returning both formats (though only documenting the "correct" format would be a way forward) though that may look confusing when people use register/debug

Reply:
Maybe in this case it would even be appropriate the just deprecate the module and write a fresh one. This was done in the past already for some AWS modules, AFAIK. Although that's for the maintaincers to decide of course.